data/themes/darktable.css

```diff
@@ -1779,6 +1779,17 @@ scale contents trough slider:hover
   background-color: @scroll_bar_active;
 }
 
+.iop_drop_before
+{
+   border: 2px solid black;
+   border-bottom: 3.5em solid black;
+}
+
+.iop_drop_after
+{
+   border: 2px solid black;
+   border-top: 3.5em solid black;
+}
```

Comment:
3.5em is really too big and make that not fine-tuned. 2.6em is for me the better much better choice  as it's just little more bigger than module height. 2.8em is the higher acceptable value here for me. On both lines of course.

For all lines, color black is not suitable on all themes. I just suggest to replace them by @plugin_bg_color that is exactly done for such a new thing.